tasks/gulp/compile-assets.mjs

```diff
@@ -98,7 +98,7 @@ compileStylesheets.displayName = 'compile:scss'
 function compileStylesheet (stream, options = {}) {
   return stream
     .pipe(plumber(errorHandler(stream, 'compile:scss')))
-    .pipe(sass(options))
+    .pipe(sass({ includePaths: ['node_modules'], ...options }))
```

Comment:
Added `node_modules` so the Review app can grab `govuk-frontend`

Dart Sass seemed to have issues with `../../../src` resolution

Reply:
Fixed in:

* https://github.com/alphagov/govuk-frontend/pull/3184